platforms/python/versionsToBuild_11172020.txt

```diff
@@ -0,0 +1,38 @@
+# version, gpg keys, dockerfile
```

Comment:
Same here. Why is the file suffixed with `_11172020`?

Reply:
I only wanted to build current versions of python ... to avoid issues with not finding unsupported old versions of python. So I kept it as backup. Anyway we have corresponding stretch sdks already available, until unless a patch is required we wont need them